src/NServiceBus.Core/DetectObsoleteConfigurationSettings.cs

```diff
@@ -26,21 +23,6 @@ protected internal override void Setup(FeatureConfigurationContext context)
 
         static void DetectObsoleteConfiguration(UnicastBusConfig unicastBusConfig)
         {
-            if (!string.IsNullOrWhiteSpace(unicastBusConfig?.ForwardReceivedMessagesTo))
-            {
-                throw new NotSupportedException($"The {nameof(UnicastBusConfig.ForwardReceivedMessagesTo)} attribute in the {nameof(UnicastBusConfig)} configuration section is no longer supported. Switch to the code API by using '{nameof(EndpointConfiguration)}.ForwardReceivedMessagesTo' instead.");
-            }
-
-            if (!string.IsNullOrWhiteSpace(unicastBusConfig?.DistributorControlAddress))
-            {
-                throw new NotSupportedException($"The {nameof(UnicastBusConfig.DistributorControlAddress)} attribute in the {nameof(UnicastBusConfig)} configuration section is no longer supported. Remove this from the configuration section. Switch to the code API by using '{nameof(EndpointConfiguration)}.EnlistWithLegacyMSMQDistributor' instead.");
-            }
-
-            if (!string.IsNullOrWhiteSpace(unicastBusConfig?.DistributorDataAddress))
-            {
-                throw new NotSupportedException($"The {nameof(UnicastBusConfig.DistributorDataAddress)} attribute in the {nameof(UnicastBusConfig)} configuration section is no longer supported. Remove this from the configuration section. Switch to the code API by using '{nameof(EndpointConfiguration)}.EnlistWithLegacyMSMQDistributor' instead.");
-            }
-
             if (unicastBusConfig?.TimeToBeReceivedOnForwardedMessages > TimeSpan.Zero)
             {
                 Logger.Error($"The use of the {nameof(UnicastBusConfig.TimeToBeReceivedOnForwardedMessages)} attribute in the {nameof(UnicastBusConfig)} configuration section is discouraged and will be removed in the next major version.");
```

Comment:
I think we need to change the log entries to throw exceptions (and adjust the message test)